_includes/navbar.html

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-toggleable-md navbar-inverse bg-faded">
+  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#myNav" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
yeah, it's a bootstrap toggle navbar thing like [this](https://getbootstrap.com/docs/3.3/examples/navbar/) but weirdish